wfapi/wfapi.ipldsch

```diff
@@ -337,13 +359,21 @@ type ItemLabel string
 type ReleaseName string
 type CatalogReleaseCID string
 
-type CatalogMirror union {
-	| CatalogMirrorByWare "byWare"
-	| CatalogMirrorByModule "byModule"
+type CatalogMirrorsCapsule union {
+	| CatalogMirrors "catalogmirrors.v1"
+} representation keyed
+
+type CatalogMirrors union {
+	| CatalogMirrorsByWare "byWare"
+	| CatalogMirrorsByModule "byModule"
 } representation keyed
 
 type WarehouseAddrList [WarehouseAddr]
 
-type CatalogMirrorByWare {WareID:WarehouseAddrList}
-type CatalogMirrorByModule {ModuleName:CatalogMirrorsByPacktype}
+type CatalogMirrorsByWare {WareID:WarehouseAddrList}
+type CatalogMirrorsByModule {ModuleName:CatalogMirrorsByPacktype}
 type CatalogMirrorsByPacktype {Packtype:WarehouseAddrList}
+
+type ReplayCapsule union {
```

Comment:
:thinking: Okay, thinking outloud for a moment:

This type isn't strictly necessary, I think?  We could just use PlotCapsule in the code relating to replays, and that would be functionally correct?

But writing it this way with the extra type can be argued to be clearer; and if we do introduce new versions of replay formation that begin to contain more than just a plot, then this type would become necessary anyway.  Putting it in now will make such a future evolution slightly more obvious to implement.  (And we have a niggling suspicion we might do such a thing.)

Yeah, okay.  Seems legit.